packages/amplication-client/src/Project/ProjectSideBar.scss

```diff
@@ -0,0 +1,17 @@
+@import "../style/index.scss";
+
+.project-sidebar {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: var(--large-spacing);
+  gap: var(--large-spacing);
+  width: 280px;
+  background: var(--surface);
```

Comment:
remove

Reply:
remove what? The background? @yuval-hazaz

Reply:
yes, there is no reason to use a background. it should be transparent and get the background from its parent